lib/cisco_node_utils/cmd_ref/bgp_af.yaml

```diff
@@ -1,41 +1,49 @@
 # bgp_af.yaml
 ---
 _template:
-  config_get: 'show running bgp all'
   config_get_token: '/^router bgp <asnum>$/'
   config_get_token_append:
     - '/^vrf <vrf>$/'
     - '/^address-family <afi> <safi>$/'
   config_set: "router bgp <asnum>"
   config_set_append:
-    - 'vrf <vrf>'
-    - 'address-family <afi> <safi>'
+    - ' vrf <vrf>'
+    - ' address-family <afi> <safi>'
+  cli_nexus:
+    config_get: 'show running bgp all'
+  cli_ios_xr:
+    config_get: 'show running-config router bgp'
 
 additional_paths_install:
```

Comment:
I think that additional_paths_install is not supported on XR at all (even under default vrf).  Can you double check?